keel-api/src/main/kotlin/com/netflix/spinnaker/keel/api/ClusterDeployStrategy.kt

```diff
@@ -6,15 +6,15 @@ import java.time.Duration.ZERO
 sealed class ClusterDeployStrategy {
   open val isStaggered: Boolean = false
   open val stagger: List<StaggeredRegion> = emptyList()
-  abstract fun toOrcaJobProperties(): Map<String, Any?>
-  abstract fun withDefaultsOmitted(): ClusterDeployStrategy
+  abstract val considerOnlyInstanceHealth: Boolean
```

Comment:
I think that `instanceHealth` could be a little confusing - because the instances aren't actually reporting health. It's the cloud provider that is telling you "these exist". I feel like as a user I'd see this property and be like "really? that's how it tell it I have no health? That's confusing.